src/main/kotlin/com/safehill/kclient/network/local/LocalServerInterfaceImpl.kt

```diff
@@ -103,6 +103,10 @@ class LocalServerInterfaceImpl : LocalServerInterface {
         TODO("Not yet implemented")
     }
 
+    override suspend fun registerDevice(deviceId: String, token: String): String {
+        TODO("Not yet implemented")
```

Comment:
This should return a not supported. I don't believe this route will ever make sense on local

Reply:
LocalServerInterfaceImpl is a dummy class... The real LocalServer lies in the Android project.

Reply:
Either way it's not "not yet implemented". There's no TODO

Reply:
I am more inclined towards removing the LocalserverInterfaceImpl altogether but its being used in the tests to create an object of ServerProxy... Shall I throw as error for now?

Reply:
Move it to the test package if so and make it throw an error so we can distinguish between not implemented and not meant to be implemented

Reply:
Refactored.